### DISCUSSION TITLE
"The importer could not connect to Firefly III" but curl works fine

### DISCUSSION THREAD

**Discussion by wsw70** [firefly-iii/firefly-iii#9153] [Firefly III]
I set up _Firefly III_ and _Firefly III Data Importer_, I can access both UIs.
The _Firefly III Data Importer_ shows an issue:

![image](https://github.com/user-attachments/assets/21fff72d-2630-4bcf-9026-b215e33f05f0)

I opened a shell **within the _Firefly III Data Importer_ container** and tried a `curl`:

```
root@fireflyiii_importer:/var/www/html# curl fireflyiii_core:8080
<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8" />
        <meta http-equiv="refresh" content="0;url='http://fireflyiii_core:8080/login'" />

        <title>Redirecting to http://fireflyiii_core:8080/login</title>
    </head>
    <body>
        Redirecting to <a href="http://fireflyiii_core:8080/login">http://fireflyiii_core:8080/login</a>.
    </body>

root@fireflyiii_importer:/var/www/html# curl http://fireflyiii_core:8080/api/v1/about
{"message":"Unauthenticated","exception":"AuthenticationException"}
```

So _Firefly III_ responds on `http://fireflyiii_core:8080` and `http://fireflyiii_core:8080/api/v1/about` (the latter is correctly unauthenticated, and responds so) - I am not sure where the problem is, as pointed out by the error message?

**Comment by wsw70** [comment]
After some time, when pressing "Start over", the connection was OK. It must have been a glitch, sorry for the trouble.

**Reply by JC5** [reply]
Strange! Keep me posted, if it happens again I want to investigate 🕵🏻
